MainWindow.xaml.cs

```diff
@@ -294,12 +329,12 @@ public void ResetRound()
             if (System.Threading.Thread.CurrentThread != Dispatcher.Thread)
             {
                 //Hand the task to the dispatcher
-                Dispatcher.BeginInvoke(new ResetRoundDelegate(ResetRound));
+                Dispatcher.BeginInvoke(new Action(ResetRound));
                 return;
             }
 
             EnableActionButtons();
-            ResultsText.Content = "A player has left! This round has reset";
+            ResultsText.Content = "A player has left!\nThis round has reset";
```

Comment:
I feel this might what to be a more intrusive message so the user knows. Maybe using the modal you made or using a popup box would be a good idea now that I think about it.

Reply:
yeah that could work too for sure. Definitely a more ideal method if we care to put in the extra time but may just say fuck it at this point on this one honestly haha

Reply:
You make a very good arguement